nixos/doc/manual/development/writing-nixos-tests.section.md

```diff
@@ -380,3 +380,24 @@ with foo_running:
     def foo_running():
         machine.succeed("pgrep -x foo")
     ```
+
+## Adding Python packages to the test script {#ssec-python-packages-in-test-script}
+
+When additional Python libraries are required in the test script, they can be
+added using the parameter `withAdditionalPythonPackages`. For example, you could add
+`numpy` like this:
+
+```py
+import ./make-test-python.nix {
+  withAdditionalPythonPackages = p: [p.numpy];
```

Comment:
I haven't seen `with*` in as a parameter before. How about `extraPythonPackages`?

Reply:
`extra` sounds much nicer, thanks for the suggestion. I've renamed the tests as well to match the new name and updated the docs.